nautobot/ipam/models.py

```diff
@@ -1043,8 +1043,7 @@ class Meta:
     def __init__(self, *args, **kwargs):
         address = kwargs.pop("address", None)
         namespace = kwargs.pop("namespace", None)
-        # We don't want users providing their own parent since it will be derived automatically.
-        parent = kwargs.pop("parent", None)
+        parent = kwargs.get("parent", None)
```

Comment:
Why the change from `pop` to `get`? Does the comment about not allowing users to directly set `parent` no longer apply because of the `clean` changes?

Reply:
Yes, because if it is popped, we will no longer be able to validate it in the clean method.